src/DTTrigGeomUtils.cc

```diff
@@ -27,6 +30,7 @@
 
 using namespace edm;
 using namespace std;
+using namespace cmsdt;
```

Comment:
Please remove the `using namespace cmsdt` statement, and write things as `HIGHLOWQ` as  `cmsdt::HIGHLOWQ`.
They are pretty non-standard variables, hence better knowing where they come from.

Reply:
Hi @battibass , should be solved in the next commit.